registry/example_const_test.go

```diff
@@ -0,0 +1,19 @@
+/*
+Copyright The ORAS Authors.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Package registry_test gives examples code of functions in the registry package.
+package registry_test
```

Comment:
Maybe` example_test_const.go` is a better filename? `example_const_test.go` is misleading.

Reply:
Both files need to be end with `_test.go` in order to be successfully compiled.